Assets/Scripts/Player/PlayerControls.cs

```diff
@@ -506,9 +503,9 @@ Vector3 GetMoveDirection()
 			inputVec *= WadeUtils.DUALINPUTMOD; // this reduces speed of diagonal movement
 		}
 
-		if ( _actor.actorCamera.cam )
+		if ( _actor.camera.cam )

```

Comment:
Can we change PlayerCamera to PlayerCameraController so this can be _actor.cameraController.camera and be less upsetting?